Client/Frontend/Home/HistoryHighlights/HighlightItem.swift

```diff
@@ -16,24 +16,6 @@ protocol HighlightItem {
     var type: HighlightItemType { get }
 }
 
-extension ASGroup: HighlightItem {
-    var type: HighlightItemType {
-        return .group
-    }
-
-    var displayTitle: String {
-        return searchTerm
-    }
-
-    var description: String? {
-        return "\(groupedItems.count)"
-    }
-
-    var siteUrl: URL? {
-        return nil
-    }
-}
```

Comment:
Since it's small enough, I liked seeing everything about ASGroup in one place. But I have no preference honestly. I'm okay with making a new file, or moving it back. 

What would you like?

Reply:
Let's make a new file. This may grow so that would be the best place for it. :)